backend/pom.xml

```
@@ -222,12 +227,12 @@
 			<plugin>
 				<groupId>org.xolstice.maven.plugins</groupId>
 				<artifactId>protobuf-maven-plugin</artifactId>
-				<version>0.5.0</version>
+				<version>0.6.1</version>
 				<configuration>
-					<protocArtifact>com.google.protobuf:protoc:3.4.0:exe:${os.detected.classifier}</protocArtifact>
+					<protocArtifact>com.google.protobuf:protoc:3.8.0:exe:${os.detected.classifier}</protocArtifact>
 					<pluginId>grpc-java</pluginId>
 					<pluginArtifact>io.grpc:protoc-gen-grpc-java:1.16.1:exe:${os.detected.classifier}</pluginArtifact>
-					<!--<protoSourceRoot>../protos</protoSourceRoot>-->
+					<protoSourceRoot>src/main/proto/protos/public</protoSourceRoot>
```

@ad-47 : let us know if this works on windows . we can add steps to reslve it inline or in readme. if required.

---

@conradoverta @ravishetye In the windows machine above path is not working. it should be like below for supporting on the windows machine.
```suggestion
<protoSourceRoot>../protos/protos/public</protoSourceRoot>
```